Resync web-platform-tests/permissions from upstream

RupinMittal opened the pull request:
#### 788c2b1a83c8551496aa04c7e32d8857032ab123
<pre>
Resync web-platform-tests/permissions from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=242377">https://bugs.webkit.org/show_bug.cgi?id=242377</a>

Reviewed by Sihui Liu.

Resync web-platform-tests/permissions from upstream.

* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/permissions/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/nfc-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/nfc-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-cg.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-cg.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-query-feature-policy-attribute.https.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-query-feature-policy-attribute.https.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/permissions/resources/empty.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/screen-wake-lock-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/screen-wake-lock-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-background-fetch-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-background-fetch-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-periodic-background-sync-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-periodic-background-sync-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/252285@main">https://commits.webkit.org/252285@main</a>
</pre>

szewai reviewed (approved):
Looks good to me! Let's wait for the bot results before landing.

RupinMittal committed:
Resync web-platform-tests/permissions from upstream
https://bugs.webkit.org/show_bug.cgi?id=242377

Reviewed by Sihui Liu.

Resync web-platform-tests/permissions from upstream.

* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/permissions/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/all-permissions.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/nfc-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/nfc-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/non-fully-active.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-cg.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-cg.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-query-feature-policy-attribute.https.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions/permissions-query-feature-policy-attribute.https.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/permissions/resources/empty.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/screen-wake-lock-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/screen-wake-lock-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-background-fetch-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-background-fetch-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-periodic-background-sync-permission-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/test-periodic-background-sync-permission.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/permissions/w3c-import.log:

Canonical link: https://commits.webkit.org/252285@main
